### PR TITLE
Adds support for COS

### PR DESCRIPTION
**Pull request by faraazkhan**
* Loads /etc/ from host as a read only volume from probe loader can determine OS
* Enables the BPF Probe Loader via Environment Variable.

Both of these are necessary for sysdig to work on COS.

Tested against the following version of COS:

```
cat /etc/os-release

BUILD_ID=10895.138.0
NAME="Container-Optimized OS"
KERNEL_COMMIT_ID=edb81c460eee7a4a35e2a95ebf6450df0963398e
GOOGLE_CRASH_ID=Lakitu
VERSION_ID=69
BUG_REPORT_URL="https://cloud.google.com/container-optimized-os/docs/resources/support-policy#contact_us"
PRETTY_NAME="Container-Optimized OS from Google"
VERSION=69
GOOGLE_METRICS_PRODUCT_ID=26
HOME_URL="https://cloud.google.com/container-optimized-os/docs"
ID=cos

```

### CODE REVIEW

**Review by leodido — APPROVED** [review]
Thanks for your first contribution!

**Comment by faraazkhan** [comment]
I can't merge this myself, please let me know if there is anything you need before this can be merged. Thanks!

**Pull request merged**
